apiary.apib

```diff
@@ -762,6 +762,39 @@ Get a list of all work types, sorted alphabetically (on their name).
                 + id: `811a5825-96f4-4318-83c3-2840935c6003` (string)
                 + name: `Planning` (string)
 
+## Templates [/templates]
+
+### templates.list [POST /templates.list]
+
+Get a list of all templates.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + department_id: `a344c251-2494-0013-b433-ccee8e8435e5` (string)
+            + type: `invoice` (enum[string])
+                + Members
+                    + delivery_note
+                    + invoice
+                    + order
+                    + order_confirmation
+                    + quotation
+                    + timetracking_report
```

Comment:
Q: Should we convert this to `time_tracking` instead? Or are there more time tracking templates?

Reply:
This refers to a report containing all the info about multiple trackings.
If we'd change this I' d rather go for `timesheet`. But we'll leave it like it is.

Reply:
Yes, let's change this!

Reply:
Any updates on this one?

Reply:
Let's keep it as is for the time being. We have no margin in our sprint for renaming this stuff. But feel free to take initiative if you want.

Reply:
Can we remove it from the documentation in the meanwhile then? Rather publish it without `timetracking_report` then having to do a BC-break later.

Reply:
And support all but one document template?

Reply:
Since we use it internally only (even just for WO) we decided to merge on the WO docs integration branch and do the necessary before releasing that one.